src/chromium/adding-third-party-crates.md

```diff
@@ -0,0 +1,25 @@
+# Adding third party crates
+
+Rust libraries are called "crates" and are found at [crates.io][0]. It's *very
+easy* for Rust crates to depend upon one another. So they do!
+
+| Property | C++ library | Rust crate |
+| --- | --- | --- |
+| Build system | Lots | Consistent - `Cargo.toml` |
+| Typical library size | Large | Small |
+| Transitive dependencies | None | Lots |
```

Comment:
nit: I would suggest replacing "None" with "Minimal" (after all `libpng` does depend on `zlib`;  it's weird that this dependency works out given that the libraries don't really know about one another, but it does...).

Reply:
Thanks, I've also added some slide notes here